src/ol_infrastructure/lib/consul.py

```diff
@@ -25,6 +30,19 @@ def build_consul_userdata(env_name):
                             ),
                             "owner": "consul:consul",
                         },
+                        {
+                            "path": "/etc/consul.d/99-autojoin-wan.json.json",
+                            "content": json.dumps(
+                                {
+                                    "retry_join_wan": [
+                                        "provider=aws tag_key=consul_env "
+                                        f"tag_value={env_name}"
+                                    ],
```

Comment:
We'll want to build up a list of strings for each entry in the wan_peer_envs (or whatever name we use) list so that the server will join up with all of them.